docs/storefront-integration/after-shopify-2.0/anatomy-of-sesami-button.md

```diff
@@ -0,0 +1,206 @@
+In certain custom themes, Sesami may face difficulty identifying the Block added in the theme editor. In such cases, we may need to manually integrate the Sesami button within a specific snippet or section. This involves removing the block from the theme editor and then inserting the following snippet of code wherever we wish to place the Sesami button.
+
+```html title="main-product.title"
+<form action="/cart/add">
+  <input type="hidden" name="id" value="{{product.selected_or_first_available_variant.id}}">
+  <input type="hidden" name="quantity" value="1">
+  <sesami-button 
+    shop-id="{{shop.id}}" 
+    product-id="{{product.id}}"
+    button-label="Select Time"
+    locale="{{ localization.language.iso_code }}"
+    button-customized-settings="%7B%22auto_add_to_cart%22%3Afalse%2C%22skip_cart%22%3Afalse%2C%22button-label%22%3A%22Select time%22%2C%22alignment%22%3A%22center%22%2C%22color%22%3A%22%22%2C%22color_background%22%3A%22%23FFF%22%2C%22font_size%22%3A16%2C%22border_width%22%3A1%2C%22border_radius%22%3A4%2C%22border_color%22%3A%22%23000%22%2C%22width%22%3A%22300px%22%2C%22height%22%3A%2245px%22%2C%22experience%22%3A%22classic%22%7D" 
+    style="align-items: center;">
+  </sesami-button>
+  <input type="submit" style="display: none">
+</form>
+```
+
+## Options
+
+### product-id
+If you want to render a button on a page different from a product page, you'll need to tell Sesami for which product you are rendering a button. Depending on your implementation, if you are rendering the button outside a product form or if a variant input is not present on the form, you might need to add a [variant-id](#variant-id) as well.
+
+```html {3}
+<sesami-button
+  .
+  product-id="XXXXXXXXXXXX"
+  .
+/>
+```
+
+### variant-id
+If you want to render a button on a page different from a product page, you'll need to tell Sesami for which variant you are rendering a button. You always need to specify the [product-id](#product-id) when using this option
+
+```html {3}
+<sesami-button
+  .
+  data-sesami-product-id="XXXXXXXXXXXX"
+  data-sesami-variant-id="XXXXXXXXXXXX"
+  .
+/>
+```
+
+### shop-id
+If you want to integrate Sesami in an external page or website, you'll need to tell Sesami for which store you want to display the button; this is done by providing the `shop-id` option:
+
+**Both [variant-id](#variant-id) and [product-id](#product-id) are required when using this option.**
+
+:::info Where can I find my shop ID?
+
+Each store has its own shop id which can be fetched by visiting this address after replacing `STORE` with your store address: http://STORE.myshopify.com/admin/shop.json
+:::
+
+```html {3}
+<sesami-button
+  .
+  shop-id="XXXXXXXXXXXX"
+  data-sesami-product-id="XXXXXXXXXXXX"
+  data-sesami-variant-id="XXXXXXXXXXXX"
+  .
+/>
+```
+
+If you need to load Sesami on an external page (like a WordPress website), you'll need to add this line of code to your page:
+
+```html
+<script async src="https://cdn.sesami.co/shopify.js"></script>
```

Comment:
This needs to be changed to the new URL:
https://cdn.sesami.dev/storefront/latest/sesami-main.js

We also need to add instructions to add the stylesheet (Similar to app extension)
Ref PR: https://github.com/sesamiapp/app-extension/pull/107/files
Ticket: https://sesami.atlassian.net/browse/SET-942